src/components/Comments/CommentItem/index.tsx

```diff
@@ -0,0 +1,143 @@
+import BoxBase from "../../Boxs/BoxBase";
+import ButtonBase from "../../Buttons/ButtonBase";
+import DividerBase from "../../Dividers/DividerBase";
+import TypographyBase from "../../Typography/TypographyBase";
+import CommentForm from "../CommentForm";
+
+import convertLocateTime from "src/utils/convertLocalTime";
+
+export interface CommentItemProps {
+    commentId: string;
+    authorName: string;
+    authorAvatar?: string | null;
+    content: string;
+    createdAt: string;
+    deleteCommentHandler: (props: { commentId: string }) => void;
+    isDeleteCommentLoading?: boolean;
+    updateCommentHandler: (props: { commentId: string; content: string }) => void;
+    isUpdateCommentLoading?: boolean;
+}
+
+export const defaultAvatarSrc =
+    "https://www.gravatar.com/avatar/d7465b8fce04fd17a376618ba1eb8ddc.jpg?d=https%3A//d2mcnjhkvrfuy2.cloudfront.net/static/images/avatar_default.png&s=64";
+
+const CommentItem = (props: CommentItemProps) => {
+    const {
+        commentId,
+        authorName,
+        authorAvatar,
+        content,
+        createdAt,
+        deleteCommentHandler,
+        isDeleteCommentLoading,
+        updateCommentHandler,
+        isUpdateCommentLoading,
+    } = props;
+
+    const [isEdit, setIsEdit] = useState<boolean>(false);
+
+    const deleteHandler = () => {
+        deleteCommentHandler({ commentId });
+    };
+    const updateHandler = (content: string) => {
+        updateCommentHandler({ commentId, content });
+        setIsEdit(false);
+    };
+
+    const commentCreatedTime = convertLocateTime({ time: createdAt });
+
+    return (
+        <Grid container alignItems="center">
+            <Grid item xs={11} sm={10} md={9}>
+                <DividerBase highlighted />
+                <BoxBase display="flex">
+                    <DividerBase orientation="vertical" flexItem highlighted />
+                    <BoxBase
+                        flexGrow={1}
+                        display="flex"
+                        flexDirection="column"
+                        ml={4}
+                        mt={3}
+                        mb={1}
+                    >
+                        <BoxBase
+                            display="flex"
+                            justifyContent="space-between"
+                            alignItems="center"
+                            pb={2}
+                        >
+                            <BoxBase display="flex" alignItems="center">
+                                <BoxBase mr={1}>
+                                    <AvatarBase
+                                        data-testid="CommentItem__avatar"
+                                        src={authorAvatar ? authorAvatar : defaultAvatarSrc}
+                                    />
+                                </BoxBase>
+                                <BoxBase display="flex" flexDirection="column">
+                                    <TypographyBase component="div" variant="body2">

```

Comment:
I don't think having a div typography is a good idea, please use <p> and span instead